### PR TITLE
Fix/welcome page for fr

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Change welcome page for FR
MBE-159

Needs #839 to be merged

### CODE REVIEW

File: client/app/configuration/configuration.controller.js [hunk 1]
```diff
@@ -1,51 +1,72 @@
...
+
+      $onInit() {
+        this.guides = this.constants.TOP_GUIDES;
+
+        this.currentGuides = [];
+        this.currentTypeOfGuide = null;
+
+        this.selectTypeOfGuide('domainHosting');
+
+        this.helpCenterURLs = Object.keys(this.constants.urls)
```
Comment (marie-j) [inline client/app/configuration/configuration.controller.js]: Won't a `mapValues` would do ? 
```suggestion
        this.helpCenterURLs = _.mapValues(this.constants.url, (value, subsidiary) => _.get(value, 'support'))
```

Comment (FredericEspiau) [reply client/app/configuration/configuration.controller.js]: No it wouldn't:
- some keys are to be ignored (they are not subsidiaries 😅)
- my object is like this:
```js
{
  FR: 'ovh.fr/support',
  DE: 'ovh.de/support,
  ...
}
```

Comment (FredericEspiau) [reply client/app/configuration/configuration.controller.js]: Done !

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

File: client/app/configuration/configuration.html [hunk 2]
```diff
@@ -28,7 +49,7 @@ <h2 class="oui-tile__title" data-ng-bind="'core_type_of_guide_' + key | translat
                             </li>
                             <li class="oui-tile__item">
                                 <div class="oui-tile__definition">
-                                    <a data-ng-href="{{ ConfigurationCtrl.allGuides }}"
+                                    <a data-ng-href="{{ $ctrl.allGuides }}"
```
Comment (marie-j) [inline client/app/configuration/configuration.html]:
```suggestion
                                     <a data-ng-href="{{:: $ctrl.allGuides }}"
```

Comment (FredericEspiau) [reply client/app/configuration/configuration.html]: Nop, loaded asynchronously and when adding `::`, it didn't work ! A bit frustrated by the way `::` works sometimes, as documentation says that `undefined` variables will be tracked until they are

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci#843.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Pull request merged**